### PR TITLE
Bunnymark example

### PR DESCRIPTION
**Pull request by kvark**

### CODE REVIEW

File: examples/bunnymark/main.rs [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+    fn resize(
+        &mut self,
+        _sc_desc: &wgpu::SwapChainDescriptor,
+        _device: &wgpu::Device,
+        _queue: &wgpu::Queue,
+    ) {
+        //empty
+    }
+
+    /// render is called each frame, dispatching compute groups proportional
```
Comment (grovesNL) [inline examples/bunnymark/main.rs:291]: Looks like these comments are leftover from boids

File: examples/bunnymark/main.rs [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+            primitive: wgpu::PrimitiveState {
+                topology: wgpu::PrimitiveTopology::TriangleStrip,
+                ..wgpu::PrimitiveState::default()
+            },
+            depth_stencil: None,
+            multisample: wgpu::MultisampleState::default(),
+        });
+
+        let texture = {
+            let img_data = include_bytes!("icon.png");
```
Comment (grovesNL) [inline examples/bunnymark/main.rs:135]: We could use the wgpu logo or something here instead

**Comment by kvark** [comment]
This is technically ready. Interestingly, using wgpu logo made it run slower for me.
I checked, and we appear to be texture unit bound on GPU, so the logo is probably just bigger than the other picture.

**Comment by kvark** [comment]
Let's add it with an idea that:
  1. we'll potentially move the examples out into a separate repo. I think this has a benefit of allowing users to copy stuff out of the examples, easier.
  2. we'd extend this example with more features, such as `TypedArena` use.

bors r=grovesNL

**Pull request merged**